package.json

```diff
@@ -13,8 +13,11 @@
     "@tanstack/react-location-devtools": "^3.4.4",
     "@tanstack/react-location-jsurl": "^3.4.4",
     "@tanstack/react-location-simple-cache": "^3.4.4",
+    "@uiw/react-textarea-code-editor": "^2.0.1",
     "@xstate/react": "^2.0.1",
     "core-js": "^3.21.1",
+    "next-remove-imports": "^1.0.6",
```

Comment:
I see a next-js plugin here, but we don't use next-js for our project. I also see this plugin used nowhere. Is there perhaps a reason you wanted to use this plugin? If not, I think you can uninstall this. `npm uninstall next-remove-imports` should do the trick.